README.md

```diff
@@ -1,57 +1,253 @@
+<%= render "form" %>
+```
+
+Then create the partial by running ```touch app/views/posts/_form.html.erb``` and move the ```<h3>Post Form</h3>``` HTML code into the new partial. Running the tests again and everything should still be passing, and now we have abstracted our form so it can be used for more than just the ```new``` action in the future.
+
+
+## Building the form in HTML
+
+Our first pass at the form will be in plain HTML, and in this reading we're not concerned with creating any records in the database, our focus is simply on the form process, so we'll simply be printing out the submitted form params on the show page.
+
+Let's create a spec for this, it's going to take a while for this to pass since we're going to be spending some time on the HTML creation process, but it's a good practice to ensure all new features are tested before the implementation code is added.
+
+```ruby
+# specs/features/post_spec.rb
+
+scenario 'new form submits content and redirects to new page and prints out params' do
+  visit new_post_path
+
+  fill_in 'post_title', with: "My post title"
+  fill_in 'post_description', with: "My post description"
+
+  click_on "Submit Post"
+
+  expect(page).to have_content("My post title")
+end
+```
+
+This is a temporary spec since we eventually will want this to form to redirect to a newly created record page, but we'll use it to ensure our form is working properly. This fails for obvious reasons, let's follow the TDD process and let the failures help build our form. The first error says that it can't find the field ```post_title```. Let's add the following HTML form items into the ```_form.html.erb``` view template:
+
+```ERB
+<h3>Post Form</h3>
+
+<form>
+  <label>Post title:</label><br>
+  <input type="text" id="post_title" name="post_title"><br>
+
+  <label>Post Description</label><br>
+  <textarea id="post_description" name="post_description"></textarea><br>
+
+  <input type="submit" value="Submit Post">
+</form>
+```
+
+In looking at both of the input elements, I'm using the standard Rails convention:
+
+* ```id``` - This will have the model name followed by an underscore and then the attribute name
+
+* ```name``` - This is where Rails looks for the parameters and stores it in a Hash. In a traditional Rails application this will be nested inside of the model with the syntax ```model[attribute]```, however we will work through that in a later lesson.
+
+Ok, so the spec was able to fill in the form elements and submit, but it's giving an error because this form doesn't actually redirect to any page, let's first update the form so that it has an action:
+
+```ERB
+<form action="<%= posts_path %>">
+```
+
+This will now redirect to ```/posts```, however we also need to add a method so that the application knows that we are submitting form data via the POST HTTP verb:
```

Comment:
Also add a note on how you know the form should submit to the `posts_path`. Revisit REST for a sec